language/stdlib/modules/libra_version.move

```diff
@@ -1,28 +1,45 @@
 address 0x0:
 
 module LibraVersion {
+    use 0x0::Association;
     use 0x0::LibraConfig;
     use 0x0::Transaction;
 
+    resource struct CapabilityHolder {
+        cap: LibraConfig::ConfigModifierPrivilege<Self::T>,
+    }
+
     struct T {
         major: u64,
     }
 
     public fun initialize() {
+        Transaction::assert(Transaction::sender() == LibraConfig::default_config_address(), 1);
 
-        LibraConfig::publish_new_config<Self::T>(T {
+        let cap = LibraConfig::publish_new_config<Self::T>(T {
             major: 1,
-        })
+        });
+
+        move_to_sender(CapabilityHolder{ cap });
     }
 
-    public fun set(major: u64) {
-        let old_config = LibraConfig::get<Self::T>(Transaction::sender());
+    public fun set(major: u64) acquires CapabilityHolder{
+        Transaction::assert(
+            Transaction::sender() == LibraConfig::default_config_address() ||
```

Comment:
Why can the config address call this?

Reply:
I think that depends on how we are planning to control who can set this flag up. IMHO we want to set up one account uniquely for each of the config so that in the future we can just replace this assert to work with one special address instead of the two well known address.